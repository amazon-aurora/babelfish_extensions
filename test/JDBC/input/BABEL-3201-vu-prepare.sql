CREATE DATABASE babel_3201_db1;
GO

CREATE SCHEMA babel_3201_sch1;
GO

CREATE TABLE babel_3201_t1( a int identity, b int);
GO

CREATE TABLE babel_3201_sch1.babel_3201_t2(a varchar(20), b int identity);
GO

INSERT INTO babel_3201_t1 VALUES (1);
GO

INSERT INTO babel_3201_sch1.babel_3201_t2 VALUES ('string 1');
GO
