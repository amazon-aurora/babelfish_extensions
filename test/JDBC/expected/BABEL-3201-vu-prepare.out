CREATE DATABASE babel_3201_db1;
GO

CREATE SCHEMA babel_3201_sch1;
GO

CREATE TABLE babel_3201_sch1.babel_3201_t2(a varchar(20), b int identity);
GO

CREATE TABLE babel_3201_t_int( a int identity, b int);
GO

CREATE TABLE babel_3201_t_tinyint( a tinyint identity(5,1), b int);
GO

CREATE TABLE babel_3201_t_smallint( a smallint identity(10,10), b int);
GO

CREATE TABLE babel_3201_t_bigint( a bigint identity(3,3), b int);
GO

CREATE TABLE babel_3201_t_numeric( a numeric identity(4,2), b int);
GO

CREATE TABLE babel_3201_t_decimal( a decimal identity(7,3), b int);
GO

CREATE TABLE babel_3201_t1( a int identity, b int);
GO

CREATE TABLE babel_3201_t2( a int, b int);
GO

INSERT INTO babel_3201_t1 VALUES (1);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_sch1.babel_3201_t2 VALUES ('string 1');
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_tinyint VALUES (5);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_tinyint VALUES (6);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_tinyint VALUES (7);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_smallint VALUES (5);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_smallint VALUES (6);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_smallint VALUES (7);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_int VALUES (5);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_int VALUES (6);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_int VALUES (7);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_bigint VALUES (5);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_bigint VALUES (6);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_bigint VALUES (6);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_numeric VALUES (5);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_numeric VALUES (6);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_numeric VALUES (7);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_decimal VALUES (5);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_decimal VALUES (6);
GO
~~ROW COUNT: 1~~


INSERT INTO babel_3201_t_decimal VALUES (7);
GO
~~ROW COUNT: 1~~


